source/numerics.tex

```diff
@@ -2143,7 +2143,7 @@
 \end{itemdescr}
 
 \begin{itemdecl}
-void seed();  
+void A::seed();
```

Comment:
This is weird. In most itemdecls we *never* say the class name. This "random" business is very idiosyncratic, though. But I'd call this change a step backwards.